waku/v2/node/peer_manager/waku_peer_store.nim

```diff
@@ -143,3 +143,6 @@ proc selectPeer*(peerStore: PeerStore, proto: string): Option[RemotePeerInfo] =
 
 proc getPeersByDirection*(peerStore: PeerStore, direction: Direction): seq[StoredInfo] =
   return peerStore.peers().filterIt(it.direction == direction)
+
+proc getDisconnectedPeers*(peerStore: PeerStore): seq[StoredInfo] =
```

Comment:
Can we rather call this something else, like `UnconnectedPeers` or `NotConnectedPeers`. Disconnected seems a bit misleading.

Reply:
fixed